### PR TITLE
Integrate range positions.

### PR DESCRIPTION
**Pull request by paulp**

### CODE REVIEW

**Comment by adriaanm** [comment]
review (also) by @huitseeker

**Comment by paulp** [comment]
PLS REBUILD/pr-checkin-per-commit@d1ef60b

**Comment by paulp** [comment]
PLS REBUILD/pr-rangepos-per-commit

File: src/reflect/scala/reflect/internal/Positions.scala [hunk 1]
```diff
@@ -57,9 +296,25 @@ trait Positions extends api.Positions { self: SymbolTable =>
     }
   }
 
+  /** Position a tree.
+   *  This means: Set position of a node and position all its unpositioned children.
```
Comment (retronym) [inline src/reflect/scala/reflect/internal/Positions.scala:300]: Just for clarity: "unpositioned immediate children"

**Comment by paulp** [comment]
I have a bunch more which I will be building on this, so if anyone has any grave issues with it sooner would be better than later, thanks.

**Comment by retronym** [comment]
LGTM

**Pull request merged**